docs/about-mono/releases/4.6.0.md

```diff
@@ -114,6 +114,102 @@ By doing so Mono runtime will not be able to output IL offsets instead of native
 
 `mono-symbolicate` can be found [here](https://github.com/mono/mono/tree/mono-4.6.0-branch/mcs/tools/mono-symbolicate)
 
+### Log Profiler v1.0
+
+This Mono release will feature version 1.0 of the
+[log profiler](/docs/debug+profile/profile/profiler/). This is the most stable
+version of the profiler to date. While there are a few new features, the focus
+has been on hardening the profiler to run on all sorts of platforms (including
+Linux, OS X, Android, and iOS) without crashing or otherwise destabilizing the
+program being profiled.
+
+#### New Features & Changes
+
+* The profiler now has support for gathering coverage data. This data can be
+  processed into XML by `mprof-report` and can then be consumed by
+  [MonoDevelop](http://www.monodevelop.com).
```

Comment:
The XML can't be consumed by anything really.

@akoeplinger Does mprof-report give a summary of coverage data at the end of its output with your hello world test? the log profiler doesn't include coverage data by default, you'll need to enable it by passing the coverage option to the log profiler